### ISSUE TITLE
Vite 2.3 security error

### ISSUE THREAD

**Issue by mirai-will** [Kiyozz/electron-esbuild#15]
I got below error after update Vite to v2.3.0

```
 [vite] Internal server error: The request url "PROJECT_PATH/node_modules/vite/dist/client/client.js" is outside of vite dev server root "PROJECT_PATH/src/renderer". 
      For security concerns, accessing files outside of workspace root is restricted since Vite v2.3.x. 
      Refer to docs https://vitejs.dev/config/#server-fsserveroot for configurations and more details.
```

fixed with this vite.config.js 

```vite.config.js
module.exports = defineConfig({
    plugins: [reactRefresh()],
    build: {
        target: 'chrome89',
    },
+    server: {
+        fsServe: {
+            root: '../../'
+        }
+    }
});
```

**Comment by Kiyozz** [comment]
These options has been added in vite server since [b4651e1](https://github.com/Kiyozz/electron-esbuild/commit/b4651e1dfec72d2fe54244aea8ddf728ba3f4ede).

This should fix this issue in the next release

**Comment by Kiyozz** [comment]
Fixed in @electron-esbuild@1.5.0
